loader/loader.go

```diff
@@ -921,15 +930,15 @@ func renameShardingSchema(query, srcSchema, dstSchema string) string {
 	return SQLReplace(query, srcSchema, dstSchema)
 }
 
-func fetchMatchedLiteral(router *router.Table, schema, table string) (targetSchema string, targetTable string) {
+func fetchMatchedLiteral(ctx *tcontext.Context, router *router.Table, schema, table string) (targetSchema string, targetTable string) {
 	if schema == "" {
 		// nothing change
 		return schema, table
 	}
 
 	targetSchema, targetTable, err := router.Route(schema, table)
 	if err != nil {
-		log.Error(errors.ErrorStack(err)) // log the error, but still continue
+		ctx.L().Error("fail to route table", zap.String("unit", "loader"), zap.Error(err)) // log the error, but still continue
```

Comment:
s/loader/load?
is `load` added in L361?

Reply:
the fish escaping from the net